### DISCUSSION TITLE
[BUG] Reaction disappears after refreshing

### DISCUSSION THREAD

**Discussion by MidlajYH** [community/community#47905] [Mobile]
https://user-images.githubusercontent.com/85442225/219894631-bd1ccdaa-f839-4554-ad87-53f668a0c26c.mp4

Device: Poco X3 Pro
Android Version: 12

Using Github Mobile Beta v1.101.0-beta (10112)

**Comment by vickki21** [comment]
IMEI (sim slot 1)

ΙΜΕΙ

354814520747367

IMEI SV 01

**Reply by MidlajYH** [reply]
?

**Comment by melvincwng** [comment]
Hey @MidlajYH, for bugs, you can report them to GitHub over here: https://support.github.com/contact/bug-report

Hope this clarifies 🙂
